spec.md

```diff
@@ -209,6 +209,23 @@ that contains both context and data).
     [RFC 2046](https://tools.ietf.org/html/rfc2046)
 * For Media Type examples see [IANA Media Types](http://www.iana.org/assignments/media-types/media-types.xhtml)
 
+### property-bucket
+* Type: <Key Value>
+* Description: A list of “key : value” pairs in flat strcuture that specify the properties of the event source.
+  The value could be a string or a JSON object. There should be no duplication of field in this property-bucket. 
+  One example usage of the property-field is to correlate multiple events involved with a serverless application.  
```

Comment:
Are properties only added when the event is created, or can they be changed/added on the event's way through the infrastructure?

Reply:
I think they can be changed/added